src/main/resources/application.yml

```diff
@@ -11,6 +11,8 @@ application:
   conceptCatalogueHost: ${FDK_BASE_HOST:https://staging.fellesdatakatalog.digdir.no}
   organizationCatalogueHost: ${ORGANIZATION_CATALOGUE_HOST:https://organization-catalogue.staging.fellesdatakatalog.digdir.no}
   catalogUriHost: ${FDK_REGISTRATION_BASE_URI:https://registrering.fellesdatakatalog.digdir.no}/catalogs
+security:
+  fdkIssuer: ${SSO_HOST:https://sso.staging.fellesdatakatalog.digdir.no}/auth/realms/fdk
```

Comment:
Denne er ikke nødvendig. Sjekk om du kan erstatte det med ordentlig security config:

```
spring:
  security:
    oauth2:
      resourceserver:
        jwt:
          issuer-uri: http://localhost:8083/auth/realms/baeldung
          jwk-set-uri: http://localhost:8083/auth/realms/baeldung/protocol/openid-connect/certs
```

Merk at vi mangker `isuer-uri` i vår config

Reply:
Ja, men vi oppretter en egen liste av validatorer til DelegatingOAuth2TokenValidator. Listen som er satt som default i spring inneholder bare validering av issuer og timestamps, for å få med audience så blir disse to initalisert på ny.

Vi kan jo legge det til som om det var standard spring, men det gir inntrykk av at JwtIssuerValidator blir konfigurert av spring. Med `security.fdkIssuer` så er det tydelig at dette er noe vi konfigurerer selv.